Source/ACE/Network/GameEvent/Events/GameEventIdentifyObjectResponse.cs

```diff
@@ -144,7 +144,9 @@ public GameEventIdentifyObjectResponse(Session session, WorldObject obj)
                 flags |= IdentifyResponseFlags.StringStatsTable;
             }
 
-            if (obj.ItemType == ItemType.Creature)
+            // if PropertyPool.NpcLooksLikeObject == true, do not treat as creature data
+            var npcLooksLikeObject = propertiesBool.FirstOrDefault(x => x.PropertyId == (uint)PropertyBool.NpcLooksLikeObject)?.PropertyValue;
```

Comment:
Could you make this a property of WorldObject/AceObject, instead of reading it directly from propertiesBool?  In general we don't want to mess with properties outside of those classes (with rare exceptions -- like when we have to send them to the client).